docs/get-started.md

```diff
@@ -4,14 +4,43 @@ title: Temporal documentation
 sidebar_label: Get started
 ---
 
+Welcome to Temporal's user documentation!
+
 Temporal reimagines state-dependent service-orchestrated application development, and so much more.
 
 ## Get started
 
-This is page will be dedicated to directing users on how to get started with Temporal.
+Ready to develop? [Install the Temporal server](install-temporal-server) and choose your SDK.
+
+→ [Go SDK](go-quick-start)
 
-This is placeholder content.
+→ [Java SDK](java-quick-start)
 
 ## Learn about Temporal
 
-etc...
+Building applications with Temporal is a paradigm shift. If you are new to Temporal, or workflow applications in general, get caught up on the [core concepts](concepts-overview) and common [use cases](use-cases-overview).
+
+Run the code samples to see how it works:
+
+→ [Go SDK samples](https://github.com/temporalio/go-samples)
+
+→ [Java SDK samples](https://github.com/temporalio/java-samples)
+
+## Best practices
+
+Want to master Temporal Workflows and recommended development techniques? Our guides will show you how.
```

Comment:
I like "master Temporal Workflows" but don't like how you've structured the sentence to have "master" apply to "recommend development techniques". While I guess it's true that you could master "recommended development techniques" I don't think this is something people aspire to.